subsys/zigbee/osif/zb_nrf_transceiver.c

```diff
@@ -228,26 +283,32 @@ zb_bool_t zb_trans_is_active(void)
 zb_bool_t zb_trans_transmit(zb_uint8_t wait_type, zb_time_t tx_at,
 			    zb_uint8_t *tx_buf, zb_uint8_t current_channel)
 {
+	struct net_buf frag = {
+		.frags = NULL,
+		.b = { .data = &tx_buf[1],
```

Comment:
Let's move the initializer to a new line, i. e.:
```
		.b = {
			 .data = &tx_buf[1],
			...
		}
```

Reply:
fixed